### PR TITLE
migrate APIService handling

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Migrate APIService handling.
We missed that during the migration.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by alvaroaleman — COMMENTED** [review]
/lgtm
/approve

File: api/codegen/reconcile/main.go [hunk 1]
```diff
@@ -93,6 +93,11 @@ func main() {
 				ImportAlias:        "admissionregistrationv1beta1",
 				ResourceImportPath: "k8s.io/api/admissionregistration/v1beta1",
 			},
+			{
+				ResourceName:       "APIService",
+				ImportAlias:        "apiregistrationv1beta1",
+				ResourceImportPath: "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1",
```
Comment (alvaroaleman) [inline api/codegen/reconcile/main.go:99]: This doesn't look right shouldn't it be below `K8s.io/api`?

/hold

Comment (mrIncompetent) [reply api/codegen/reconcile/main.go:99]: Nope, APIServices belong to the kube-aggregator package

**Pull request merged**